snappy-c.h

```diff
@@ -68,6 +69,7 @@ typedef enum {
  *   }
  *   free(output);
  */
+SNAPPY_PUBLIC_API
```

Comment:
Is it possible to remove all SNAPPY_PUBLIC_API and use WINDOWS_EXPORT_ALL_SYMBOLS instead?